src/data/real/basic.lean

```diff
@@ -481,6 +481,29 @@ begin
   { exact csupr_const },
 end
 
+lemma exists_of_le_supr_subtype {α : Type*} {p : α → Prop}
```

Comment:
I don't think this has anything to do with reals: it should be true in any linear order, right?